README.md

```diff
@@ -1,70 +1,177 @@
+### Install
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+In the terminal, go to your file directory and run this command.
 
-### Deployment
+```
+mkdir motorcycle-app
+cd motorcycle-app
+$ git clone https://github.com/jonas-45/motorcycle-booking-react.git 
+```
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+### Usage
 
-### `npm run build` fails to minify
+### Run locally
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+```
+$ cd motorcycle-booking-react
+```
+
+Install the required dependencies to run the project with this command:
+
+```
+$ npm install
+```
+
+Then run it in your browser with this command:
+
+```
+$ npm start -p 3001
+```
+
+Kindly modify the files as needed.
+
+### Run tests
+
+To run tests, please run this command
+
+```
+$ npm test
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Authors <a name="authors"></a>
+
+👤 **Jawad Nazari**
+
+- GitHub: [@Jawad-Nazari](https://github.com/Jawad-Nazari)
+- Twitter: [@Nazari_jawad58](https://twitter.com/Nazari_Jawad58)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/jawad-nazari/)
+
+👤 **Patrick Mukula**
+
+- GitHub: [@Patfarmurs](https://github.com/Patfarmurs)
+- Twitter: [@MukulayengeP](https://twitter.com/MukulayengeP)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/Patrick-Mukula/)
+
+👤  **Wullah Jonas Nuta**
+
+- GitHub: [@jonas-45](https://github.com/jonas-45)
+- Twitter: [@jonaswullah](https://twitter.com/jonaswullah)
+- LinkedIn: [jonas-wullah](https://linkedin.com/in/jonas-wullah)
+
+👤 **Ahmad Saeed**
+
+- Github: [GitHub](https://github.com/ehmaddd/)
+- Twitter: [Twitter](https://twitter.com/ehmaddd_pk)
+- Linkedin: [LinkedIn](https://www.linkedin.com/in/ehmaddd/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭Future Features <a name="future-features"></a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/jonas-45/motorcycle-booking-react/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Show your support  <a name="️support"></a>
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments <a name="acknowledgements"></a>
+
+- [Microverse](https://www.microverse.org/)
+- Original design idea by [Murat Korkmaz on Behance](https://www.behance.net/muratk).
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/jonas-45/motorcycle-booking-react/blob/LICENSE.md) licensed.
```

Comment:
Hi @Patfarmurs ,
The link to the license file is not working. Please update it.
![image](https://github.com/jonas-45/motorcycle-booking-react/assets/115032263/8265c9ae-46e6-4b44-9ab2-b3620423e301)